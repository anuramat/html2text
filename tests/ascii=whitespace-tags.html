<!doctype html>
<html xmlns="http://www.w3.org/1999/xhtml" lang="en-US">
<body>
	<foo/>
	<?xml:namespace prefix="o" ns="urn:schemas-microsoft-com:office:office" />
<b>a </b>test!
<b> a</b>test!
<b>a
</b>test!<br>
<br>
  test
</body>
</html>
