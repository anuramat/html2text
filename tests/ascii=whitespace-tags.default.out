aa test! aatest! aa test!

test
